src/policy/policy.h

```diff
@@ -98,7 +98,7 @@ bool IsStandardTx(const CTransaction& tx, bool permit_bare_multisig, const CFeeR
      * @param[in] mapInputs    Map of previous transactions that have outputs we're spending
      * @return True if all inputs (scriptSigs) use only standard transaction forms
      */
-bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs);
+bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs, bool taproot_active = false);
```

Comment:
```suggestion
bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs, bool taproot_active);
```

I don't really like modifying code to accommodate tests. This is just asking for a mishap down the road, and at best someone else will have to clean it up anyway. It seems safer to have all callers pass down the bool explicitly.

Reply:
Ok, assuming that taproot activates, this code will go away either way, so consider my comment a style-nit.

Reply:
I've made the argument mandatory.